src/GraphQLParser.Tests/Visitors/SDLPrinterFromParsedTextTests.cs

```diff
@@ -924,11 +924,11 @@ public async Task SDLPrinter_Should_Print_BlockStrings(int number, string input,
         var document = (input + " scalar a").Parse();
 
         var printer = new SDLPrinter();
-        await printer.PrintAsync(document, writer).ConfigureAwait(false);
+        await printer.PrintAsync(document, writer);
         var renderedOriginal = writer.ToString();
 
-        var lines = renderedOriginal.Split(Environment.NewLine);
-        var renderedDescription = string.Join(Environment.NewLine, lines.SkipLast(1));
+        var lines = renderedOriginal.Split(new string[] { Environment.NewLine }, StringSplitOptions.None);
```

Comment:
I had to change signatures because of net462 TFM that lacks of previous signatures.